CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
-# 0.5.4
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.5.5] - 2022-01-12
```

Comment:
Feels like this introduces new functionality, though I'm less certain how to define that for this than the microsim
```suggestion
## [0.6.0] - 2022-01-12
```

Reply:
Yeah OK agreed, bumped minor